index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/responsive.css" />
+    <link
+      href="https://unpkg.com/boxicons@2.0.9/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div id="container">
+      <header>
+        <nav>
+          <p id="logo">Lisandro Seia</p>
```

Comment:
- [ ] The logo text should be implemented using an `a` tag. If you look at most sites you will notice that clicking on the logo on the page takes you to the homepage. Implementing this on your already fantastic website will make it really impressive 🤗. 
something like this would be really nice.
```suggestion
          <a href="/" id="logo">Lisandro Seia</a>
```